CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Support both explicit kubeconfigs and file paths.
+
+### Changed
+
+- Reduce interval when waiting for apps now app-operator has a status webhook.
```

Comment:
I think it isn't a breaking change :)

```suggestion
- Optimize apps wait interval as app-operator has a status webhook.
```

Reply:
True :) that is much nicer and I changed the heading to Fixed. PTAL